includes/class-sensei-course-structure.php

```diff
@@ -5,6 +5,9 @@
  * @package sensei
  */
 
+use Sensei\Admin\Content_Duplicators\Lesson_Quiz_Duplicator;
```

Comment:
I think we need a cleanup to remove all the things we're not using anymore, right?

Reply:
I think we need to also remove some classes and tests, right? Or are you keeping it for the future when we need it? If yes, maybe we could add it to a draft PR. WDYT?